web-app/README.md

```diff
@@ -1,3 +1,195 @@
+* `kiosk_content_version` - a version number for the file specified by `kiosk_content`. If the content's updated, this number can be incremented to cause the Python script to download the new content.
+* `kiosk_download_time` - the hour of the day for the Python script to check for updates to the `kiosk_content` env var. It can also be set to `now` to download as soon as new content is detected either by changing the `kiosk_content` variable or the `kiosk_content_version` number.
+* `kiosk_data` - An optional JSON object that can be used to overwrite the contents of the `data.js` in the ZIP file. 
+
+Keep in mind, that however the code in `data.js` is structured, the JSON in this `kiosk_data` environment variable must mimic it. For example, in the code, `data.js` looks like this:
+
+```javascript
+var data = {
+  cities: ["Barcelona", "Madrid", "Dublin", "Tokyo"],
+};
+```
+
+So for the `kiosk_data` Notehub environment variable, I created the following JSON object:
+
+```json
+{"cities": ["San Fernando", "Port of Spain", "Atlanta", "Shen Zhen"]}
+```
+
+When the project's Python script runs on the Raspberry Pi, it will read this variable from Notehub and prepend `var data=` to the front of the list, so it can sub in for the `data.js` file in the project.
+
+Here's what this demo project's Notehub env vars look like all together.
+
+![Notehub fleet env vars for the cellular-connected electronic kiosk](images/notehub-env-vars.png)
+
+## Initiate the Download of the ZIP to the Raspberry Pi
+
+Now it's time to download the ZIP file to the Raspberry Pi via the Notecard's Python script.
+
+Following the instructions in the [`script/` directory's `README.md`](../scripts/README.md):
+
+1. Clone this repo to the Raspberry Pi from GitHub, 
+2. Ensure the Pi's version of Python is 3.8 or above, 
+3. Open the repo with VS Code or the IDE of your choice, and 
+4. Type the following command into a terminal at the root of the project substituting your own `data-dir`, `product`, and `route`.
+
+```shell
+python kiosk.py --data-dir ~/kiosk-data/ --product com.blues.cellular_XXXXXXXXX_XXXX --route kiosk 
+```
+
+This should initiate the download of the ZIP file from the AWS bucket via Notehub. Depending on the size of the ZIP, the download to the Pi may take a few minutes, but once it's finished, it will automatically open a Chromium web browser displaying the web app.
+
+## Update the Notehub Env Vars and See the Changes Take Effect in the App
+
+At this point, you may have noticed that the cities being displayed in the web app on the Pi are not the same ones contained within the `data.js` file in the ZIP. This is because the Notehub environment variable `kiosk_data` is overriding the list of cities contained within the `data.js` file. `kiosk_data` acts as a way of passing dynamic data to the web app without requiring a new ZIP file to be uploaded to the AWS bucket and downloaded by the Notecard.
```

Comment:
Ok, I think you answer my last comment here:

> This is because the Notehub environment variable `kiosk_data` is overriding the list of cities contained within the `data.js` file.

That may be worth mentioning earlier where I got confused, but up to you.

Reply:
Good point, I'll make mention of it above as well